src/fswAlgorithms/_GeneralModuleFiles/dynamicModels.cpp

```diff
@@ -24,8 +24,10 @@ DynamicsModel::DynamicsModel() = default;
 DynamicsModel::~DynamicsModel() = default;
 
 /*! Call the propagation function for the dynamics
-   @param StateVector inputState
-   @param StateVector propagatedState
+   @param std::array<double, 2> interval : time interval
+   @param StateVector state : initial state
+   @param double dt: time step
+   @return StateVector propagatedState
 */
 StateVector DynamicsModel::propagate(const std::array<double, 2> interval, const StateVector& state, const double dt) const {
     double t_0 = interval[0];
```

Comment:
:large_orange_diamond: It's not in this commit but can line 38 in `dynamicsModels.cpp` be changed to call the static function with `DynamicsModel::rk4` rather than through the instance `this->rk4`?

Reply:
I'll do it here, feels like its appropriate